tests/test_clog.py

```diff
@@ -91,6 +91,16 @@ def test_multi_day(self):
             content = self._open_and_remove(log_filename)
             assert content == complete_line
 
+    def test_cant_open_stream(self):
```

Comment:
Interesting idea. I don't think that it's really needed to assert that output. I mostly just wanted coverage to make sure my code didn't crash.